site/content/contribute/more-info/webapp/unit-testing.md

```diff
@@ -61,10 +64,10 @@ Below is a brief guide on how to do component testing:
     };
 
     test('should call updateSection on handleExpand', () => {
-        // Jest mocks are automatically cleared between tests, so they don't need to be redefined.
-        const wrapper = mountWithIntl(<EmailNotificationSetting {...baseProps}/>);
+        const {getByRole} = render(<EmailNotificationSetting {...baseProps}/>);
+        const button = getByRole('button', { name: /expand/i });
 
-        wrapper.instance().handleExpand();
+        fireEvent.click(button);
```

Comment:
``userEvent`` is the preferred way for user action events

Reply:
+1, it's also mentioned here - https://testing-library.com/docs/react-testing-library/migrate-from-enzyme#simulate-user-events